satpy/readers/eps_l1b.py

```diff
@@ -144,7 +144,8 @@ class EPSAVHRRFile(BaseFileHandler):
     sensors = {"AVHR": "avhrr-3"}
 
     units = {"reflectance": "%",
-             "brightness_temperature": "K"}
+             "brightness_temperature": "K",
```

Comment:
Just because I feel like being a pain today...if this had ended with a comma and the `}` was on its own line, it wouldn't have been part of the diff. The history...the git history!!! :scream:

Reply:
Ah yes syntax diffs are nice. In the "read one more line" argument, why do you need to know the dictionary is closed (ignoring that it is just one more line)? You are going to keep reading the code to see where the dictionary is used anyway.

Reply:
Yes, that's why we don't have it 😀